src/devices/vdml_rotation.c

```diff
@@ -73,6 +75,26 @@ int32_t rotation_set_reversed(uint8_t port, bool value) {
 	return_port(port - 1, 1);
 }
 
+int32_t rotation_init_with_reverse_flag(uint8_t port, bool reverse_flag) {
```

Comment:
I know originally we wanted an initialization function, but I think a more accurate description would be "rotation_set_reversed_blocking" and update the header accordingly.